etc/stack-schema-map.yaml

```diff
@@ -34,3 +34,7 @@
 "8.2.0":
   beats: "main"  # TODO: update this once beats releases
   ecs: "1.12.1"
+
```

Comment:
We should remove the other `update this once beats releases` comments and make sure the versions are correct for older releases.

Reply:
we can refresh the ones that are available and update in a separate pr 👍